src/Tasks/ResolveManifestFiles.cs

```diff
@@ -82,6 +84,15 @@ public ITaskItem[] NativeAssemblies
             set => _nativeAssemblies = Util.SortItems(value);
         }
 
+        // Runtime assets for self-contained deployment from .NETCore runtime pack
+        public ITaskItem[] RuntimePackAssets { get; set; }
```

Comment:
I just fixed a bug for self-contained publish recently. Due to another fix touched it after all tests passing, so I am not confident about our existing test coverage.